src/one_dragon/base/notify/push.py

```
@@ -566,28 +566,124 @@ def send_mpnews(self, title, message, media_id, touser="@all"):
-        ).json()
 
-        if response["errcode"] == 0:
-            self.log_info("企业微信机器人推送成功！")
-        else:
-            self.log_error("企业微信机器人推送失败！")
+        # 1. 先发文字
+        try:
+            text_data = {"msgtype": "text", "text": {"content": f"{title}\n{content}"}}
+            resp = requests.post(url, data=json.dumps(text_data), headers=headers, timeout=15).json()
+            if resp and resp.get("errcode") == 0:
+                self.log_info("企业微信机器人文字推送成功！")
```

Ruff (0.12.2)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)